testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayViewModel.kt

```diff
@@ -116,6 +116,7 @@ class MiniAppDisplayViewModel constructor(
         lifecycle.addObserver(miniAppDisplay)
     }
 
+    //NOSONAR
```

Comment:
@shohiebsense, //Using NOSONAR is not recommended. But if you still want to use it, please add why this is added

Reply:
alright, proposing another approach.